### ISSUE TITLE
resolve -t completeness issue

### ISSUE THREAD

**Issue by wisechengyi** [coursier/coursier#661]
Hi,

I've noticed that when two artifacts with share a set of dependencies, `resolve -t` will not show one of them.
```
$ ./coursier resolve -t org.apache.avro:avro:1.7.4
  Result:
└─ org.apache.avro:avro:1.7.4
   ├─ com.thoughtworks.paranamer:paranamer:2.3
   ├─ org.apache.commons:commons-compress:1.4.1
   │  └─ org.tukaani:xz:1.0
   ├─ org.codehaus.jackson:jackson-core-asl:1.8.8
   ├─ org.codehaus.jackson:jackson-mapper-asl:1.8.8
   │  └─ org.codehaus.jackson:jackson-core-asl:1.8.8
   ├─ org.slf4j:slf4j-api:1.6.4
   └─ org.xerial.snappy:snappy-java:1.0.4.1
```
which includes
```
   ├─ org.apache.commons:commons-compress:1.4.1
   │  └─ org.tukaani:xz:1.0
```

but if I do 
```
./coursier resolve -t org.apache.avro:avro:1.7.4 org.apache.commons:commons-compress:1.4.1
 Result:
├─ org.apache.avro:avro:1.7.4
│  ├─ com.thoughtworks.paranamer:paranamer:2.3
│  ├─ org.apache.commons:commons-compress:1.4.1
│  │  └─ org.tukaani:xz:1.0
│  ├─ org.codehaus.jackson:jackson-core-asl:1.8.8
│  ├─ org.codehaus.jackson:jackson-mapper-asl:1.8.8
│  │  └─ org.codehaus.jackson:jackson-core-asl:1.8.8
│  ├─ org.slf4j:slf4j-api:1.6.4
│  └─ org.xerial.snappy:snappy-java:1.0.4.1
└─ org.apache.commons:commons-compress:1.4.1
          // why is there nothing over here?
```
the root level `org.apache.commons:commons-compress:1.4.1` does not have its dependencies printed out. 
1) Is that expect?
2) If yes, can I force coursier to print out all transitive dependencies from the roots specified? 

Thanks,
Yi

**Comment by wisechengyi** [comment]
which is also inconsistent with the following result that root will have the transitive dependencies.

```
$ ./coursier resolve -t -n 12  net.java.dev.jets3t:jets3t:0.9.0   
  Result:
└─ net.java.dev.jets3t:jets3t:0.9.0
   ├─ com.jamesmurty.utils:java-xmlbuilder:0.4
   ├─ commons-codec:commons-codec:1.4
   ├─ commons-logging:commons-logging:1.1.1
   ├─ org.apache.httpcomponents:httpclient:4.1.2
   │  └─ org.apache.httpcomponents:httpcore:4.1.2
   └─ org.apache.httpcomponents:httpcore:4.1.2
```
```
$ ./coursier resolve -t -n 12  net.java.dev.jets3t:jets3t:0.9.0   org.apache.httpcomponents:httpclient:4.1.2
  Result:
├─ net.java.dev.jets3t:jets3t:0.9.0
│  ├─ com.jamesmurty.utils:java-xmlbuilder:0.4
│  ├─ commons-codec:commons-codec:1.4
│  ├─ commons-logging:commons-logging:1.1.1
│  ├─ org.apache.httpcomponents:httpclient:4.1.2
│  │  └─ org.apache.httpcomponents:httpcore:4.1.2
│  └─ org.apache.httpcomponents:httpcore:4.1.2
└─ org.apache.httpcomponents:httpclient:4.1.2
   ├─ commons-codec:commons-codec:1.4
   ├─ commons-logging:commons-logging:1.1.1
   └─ org.apache.httpcomponents:httpcore:4.1.2
```

**Comment by wisechengyi** [comment]
HI @alexarchambault , I put up a PR https://github.com/coursier/coursier/pull/671 for this. It basically will output the complete tree transitively. Given it was excluding the jars that already been printed, was that intended and should I add it behind an option?

**Comment by wisechengyi** [comment]
fixed by 07e985bffca5b796602d18d20dba8826f69fd0d7
